plugins/lookup/passwordstore.py

```diff
@@ -209,10 +212,15 @@ def check_pass(self):
             ).splitlines()
             self.password = self.passoutput[0]
             self.passdict = {}
-            for line in self.passoutput[1:]:
-                if ':' in line:
-                    name, value = line.split(':', 1)
-                    self.passdict[name.strip()] = value.strip()
+            try:
+                values = yaml.safe_load('\n'.join(self.passoutput[1:]))
+                for key, item in values.items():
+                    self.passdict[key] = item
+            except yaml.YAMLError:
```

Comment:
Maybe it's better to have an `except Exception` here; for example, if the YAML isn't a dict but a list, value or something else, the above will fail with other exceptions.

Reply:
Hm - or I just catch an `AttributeError` in case the parsed value does not implement `items()`.

I feel any other error would be quite surprising and I wouldn't like to just silence it with a generic catch.

Reply:
True, `AttributeError` also works. I've checked some common types I could think of, for them it always results in `AttributeError`.